### ISSUE TITLE
Different behaviours of different OS for a same cli command

### ISSUE THREAD

**Issue by plantain-00** [palantir/tslint#1226]
### Bug Report

- __TSLint version__: `3.9.0`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
(anything that is wrong, I expect tslint discover this error)
```

#### Actual behavior

If I run `tslint 'a.ts'`:

1. for windows, it doesn't show the error
2. for mac OS X, it shows the error.

If I run `tslint "a.ts"`

1. for windows, it show the error
2. for mac OS X, it shows the error.

#### Expected behavior

I think they should have a same behaviour.

**Comment by JKillian** [comment]
Hi @plantain-00, this seems like the same issue described [here](https://github.com/palantir/tslint/issues/1091#issuecomment-206956050) in #1091. Single quotes aren't recognized by `cmd.exe`, so that's most likely what's causing you issues. You could try switching to PowerShell, which I believe allows the use of single quotes as quotes. Hope this helps!
